### DISCUSSION TITLE
Is there something like `:LspInfo` in neovim?

### DISCUSSION THREAD

**Discussion by sambigelow** [helix-editor/helix#8614] [Q&A]
I'm trying to see if any of my LSPs successfully attached, and I can't figure out how to tell. It seems like they're not, since one is supposed to be formatting the file and its not really doing anything

**Comment by calebgasser** [comment]
What you might be looking for is `hx --health [CATEGORY]`. You can can leave off the category to get a full list of them all or specify the language.

**Reply by sambigelow** [reply]
That's not quite the same. I want to see, when I'm looking at a JavaScript file, if the prettier LS is attached to that specific file
